modules/nw-ingress-controller-nodeportservice-projects.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/nw-configuring-ingress-controller-endpoint-publishing-strategy.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-controller-nodeportservice-projects_{context}"]
+= Adding a single NodePort service to an Ingress Controller
+
+Instead of creating a `NodePort`-type `Service` for each project, you can create a single service and apply it to all of your projects. You can configure this type of environment by setting your Ingress Controller to use the `NodePortService` endpoint publishing strategy.
+
+Consider this configuration for your Ingress Controller when you want to apply a set of routes, through Ingress sharding, to a node even if the node gets displaced. 
+
+Before you `NodePort`-type `Service` for each project, read the following considerations:
+
+* You must create a wildcard DNS record for the Ingress Controller.
+* You must create a route service and specify the hostname of your Ingress Controller in this service.
+* must preserve the non-default port in the curl in order to access, for example: curl <route>:<port> // what does this mean?
+* Ensure that Ingress Controller routes can access your {product-title} cluster through the host IP address assigned to each node.
+
+.Prerequisites
+
+* You installed the {oc-first}.
+* Logged in as a user with `cluster-admin` privileges.
+* You created a wildcard DNS record.
+// https://docs.openshift.com/container-platform/4.16/networking/ingress-controller-dnsmgt.html (does not detail how to create the DNS)
```

Comment:
I cannot find decent doc examples of creating such a record. Maybe it's too trivial to document this level of detail?

Reply:
wildcard DNS is described here in the Routes row: https://access.redhat.com/solutions/7024856.  I don't know of anywhere we say more than that.  It's on the admins to know this stuff.  FYI in a coredns file, a wildcard looks like this:

*.apps.cluster.example.com.   IN     A        192.168.123.11

Reply:
Thanks. Linking to KBase articles is a bit of an issue as they're maintainence-level can vary.